proto/dymensionxyz/dymension/rollapp/rollapp.proto

```diff
@@ -75,6 +74,11 @@ message Rollapp {
   // The LastStateUpdateHeight HUB height when the last state update was
   // received
   int64 last_state_update_height = 18;
+
+  // The revision number of the rollapp. starts always with 0 revision
+  uint64 revision_number = 19;
+  // The first height of the rollapp when the revision started
+  uint64 revision_start_height = 20;
```

Comment:
faulty drs missing?

Reply:
we will use the queries to get obsolete drs so we don't need to add it here